components/bl-carousel-component/src/index.js

```diff
@@ -102,8 +102,8 @@ export default function Carousel({ component, eventHandlers }) {
       onMouseLeave={ onMouseLeave }>
       { withControls && (
         <div className="carousel__controls-button">
-          <PrevButton disabled={ !!animation } onPrevButton={ onPrevButton }/>
-          <NextButton disabled={ !!animation } onNextButton={ onNextButton }/>
+          <PrevButton disabled={ !!animation } onPrevButtonClick={ onPrevButtonClick }/>
```

Comment:
Component with name PrevButton mustn't know about onPrevButtonClick action. He should know what to do when onClick triggered.
```suggestion
          <PrevButton disabled={ !!animation } onClick={ onPrevButtonClick }/>
```